.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*   @loadsmart/platform-backend
```

Comment:
Shouldn't @rcmachado be in this? He is the sole contributor to this project

Reply:
I just created it - but it belongs to platform.